src/components/cards/BorrowAmountCard.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+
+import { div, mul } from 'biggystring'
+import * as React from 'react'
+import { View } from 'react-native'
+import FastImage from 'react-native-fast-image'
+
+import { DECIMAL_PRECISION } from '../../util/utils'
+import { type Theme, cacheStyles, useTheme } from '../services/ThemeContext.js'
+import { EdgeText } from '../themed/EdgeText.js'
+import { Card } from './Card'
+
+export const BorrowAmountCard = ({
+  dAppIcon,
+  fiatCode,
```

Comment:
Change the name into 'currencyCode' since it might be used for crypto amount in the future.